census/estimator/trainer/model.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2016 Google Inc. All Rights Reserved. Licensed under the Apache
+# License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License. You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import multiprocessing
+
+import tensorflow as tf
+from tensorflow.contrib import layers
+from tensorflow.contrib.learn.python.learn.utils import input_fn_utils
+
+tf.logging.set_verbosity(tf.logging.INFO)
+
+CSV_COLUMNS = ['age', 'workclass', 'fnlwgt', 'education', 'education_num',
+           'marital_status', 'occupation', 'relationship', 'race', 'gender',
+           'capital_gain', 'capital_loss', 'hours_per_week', 'native_country',
+           'income_bracket']
+
+LABEL_COLUMN = 'income_bracket'
+
+DEFAULTS = [[0], [''], [0], [''], [0], [''], [''], [''], [''], [''],
```

Comment:
100 column lines should help keep this in one line.
Its also useful to think about how you have comment or structure the CSV_COLUMNS variable alongside to be able to see how the defaults correspond to the columns. This stuff needs to be understandable -- why 0 in some places and '' in some others.

Secondly, I thought you'd need something like 0.0 where there is a non-integral column.